dashboard/src/t5gweb/libtelco5g.py

```diff
@@ -549,6 +549,143 @@ def generate_stats(account=None):
 
     return stats
 
+def generate_user_stats(engineer=None):
```

Comment:
rather than have a new function, could you modify `generate_stats` to take engineer as optional? similar to how you updated `generate_histogram_stats` and `get_new_comments`

Reply:
you are right, I did miss this one, thanks!